client/client.go

```diff
@@ -349,19 +349,22 @@ func (client *Client) SysInfo() (*SysInfo, error) {
 
 // CreateUserResult holds the result of a user creation
 type CreateUserResult struct {
-	Username string `json:"username"`
+	Username    string `json:"username" yaml:"username"`
+	SshKeyCount int    `json:"ssh_key_count" yaml:"ssh_key_count"`
 }
 
 // createUserRequest holds the user creation request
 type createUserRequest struct {
```

Comment:
Let's please expose this as CreateUserOptions, and take that as the parameter of CreateUser as a pointer. This will easily grow more fields.
.